app/Providers/CustomValidationProvider.php

```diff
@@ -59,6 +59,34 @@ public function boot()
             return str_replace(':other', $parameters[0], $message);
         });
 
+        /**
+         * Is the supplied voucher code value greater than the the target voucher code value
+         */
+        Validator::extend('codeGreaterOrEqual', function ($attribute, $value, $parameters, $validator) {
+            // Grab the regex matched array
+            $val = Voucher::splitShortcodeNumeric($value);
+
+            // Grab the content of the parameter we pass the rule in a roundabout fashion
+            $otherVal = array_get(
+                $validator->getData(),
+                $parameters[0]
+            );
+
+            if (!empty($otherVal)) {
+                $other = Voucher::splitShortcodeNumeric($otherVal);
```

Comment:
we should be more descriptive with the variable names, no?

Reply:
Was straight copy of the block above. If you decide to change, please change there also.

Reply:
done